KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment2/BasicStockService.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.KaylaKornelis.advancedjava.Assignment2;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+/**
+ *
```

Comment:
comments are important and mostly missing in this file. 
But as a basic implementation this is good.